hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -13,13 +13,15 @@ filesets:
       - lowrisc:prim:clock_div
       - lowrisc:prim:clock_gating
       - lowrisc:prim:clock_inv
-      - lowrisc:prim:prim_pkg
       - lowrisc:prim:lc_dec
       - lowrisc:prim:lfsr
       - lowrisc:ip:pinmux_reg
       - lowrisc:ip:pinmux_component
+      - lowrisc:prim:prim_pkg
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:edn_pkg
+      - lowrisc:systems:rstmgr_pkg
```

Comment:
technically you should use `lowrisc:ip:rstmgr` here and let the top level redirects, but it doesn't REALLY matter.

Reply:
`systems:rstmgr_pkg` will need to be changed to `ip:rstmgr_pkg` later.  There's some dependency related to how the cw305 fpga is done, but i can do that later as a different PR.

Reply:
Will fix it land test it before I will update the core file.